llvm/projects/evm-sdk/evm-stdlib/stdlib.s

```diff
@@ -226,6 +232,20 @@ modpow_cont:
     # Stack: [result, return_address]
     JUMP
 
+# This function registers a function(typically destructor) to be called by exit
+# Arguments:
+# - address of the function to be called
+# - argument that will be passed to the function
+# - dso_handle
+# Does nothing for a now.
+STDLIB_FUNC __cxa_atexit
```

Comment:
Why don't we implement `__cxa_atexit` in C?

Reply:
Since it does nothing at the moment, the assembler version is simpler and more optimal.